### PR TITLE
Yin - Working on Issue #12

### LINKED ISSUE #16: Issue 12: Add a "README.md" file for "In-Browser Execution for Compiler Explorer" project (CE frontend)
The new version of "README.md" file will include information about adding Cflat language, adding Cflat compiler, how to compile it from Wasm instead of CE server, how to add another compiler, how to run different passes of a chosen compiler. Also, the build process, test run, and what to expect when running it.

### PR DESCRIPTION
**Pull request by geniuseva**
1) Add the `In-browserExecution.md` file under the `docs/` directory, which contains our CS 590 project info.
2) edit the main's `README.md` file to add a link to our `In-browserExecution.md` file

### CODE REVIEW

File: docs/In-browserExecution.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# In-browser Execution for Compiler Explorer
+
+## What is it?
+It is an enhancement feature of Compiler Explorer (CE) by adapting it to support in-browser compilers. The primary objectives include restructuring the backend by removing the original CE server and integrating WebAssembly (Wasm) code for compiler execution. Additionally, the project aims to add a new feature of running different compiler passes so that users can better understand the compiler process. Furthermore, the project implements another functionality to load different compilers.
+
+## Build process
+
+### Installing Prerequisites
```
Comment (ashimapanjwani) [inline docs/In-browserExecution.md:13]: Why are we installing node and npm using pip (which is a python dependency manager)? Why not just do brew install node for macOS

Comment (geniuseva) [reply docs/In-browserExecution.md:13]: I changed it!

Comment (ashimapanjwani) [reply docs/In-browserExecution.md:13]: And npm gets installed with node so you don't have to install it separately.

Comment (ashimapanjwani) [reply docs/In-browserExecution.md:13]: Can you also mention that brew is only for macOS. For windows or some other OS maybe add a link from the internet that explain how to do the installation

**Review by ashimapanjwani — CHANGES_REQUESTED** [review]

**Comment by geniuseva** [comment]
@ashimapanjwani I changed again, may you check?

**Review by ashimapanjwani — APPROVED** [review]

**Comment by geniuseva** [comment]
@maemre I have made changes based on Ashima's suggestion, how do you think my changes on both files?

**Pull request merged**